test/test.issue.js

```diff
@@ -32,4 +32,24 @@ describe('Github.Issue', function() {
          });
       });
    });
+   it('should edit the issues title', function(done){
+       issues.list({}, function(err, issuesList) {
+         issues.edit(issuesList[0], {
+             title: 'Edited test title'
+         }, function(err, res) {
+            should.not.exist(err);
+            res.title.should.equal('Edited test title');
+            done();
+         });
+      });
+   });
+   it('should edit the issues title', function(done){
+       issues.list({}, function(err, issuesList) {
+         issues.get(issuesList[0], function(err, res) {
+            should.not.exist(err);
+            res.title.should.equal('Edited test title');
```

Comment:
The test should assert that the number is correct, not be based on the previous test. Tests must always be independent.